### ISSUE TITLE
Add publisher/journal: AAAS

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2283]
**Name to appear on membership page:** 
American Association for the Advancement of Science

**Payment plan:**
Membership fee + quarterly volume-based DPCs (like Wiley)

**Journals included:** 
Science
Science Advances
Science Immunology
Science Robotics
Science Signaling
Science Translational Medicine

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 
TBD

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
TBD

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**

**Email contacts to discuss integration of journal metadata with Dryad:**
TBD

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)
@mariapraetzellis @ryscher @sfisher @ahamelers

**Comment by ryscher** [comment]
Additional concerns:
- adjust `tiered_journal_reports` to use the `top_level_org` instead of `sponsor_id`, so AAS journals don't break up the calculation for Wiley journals
- adjust`tiered_journal_reports` to calculate the quarterly base count before calculating the actual billing for the quarter, so calculations match up with the [billing calculater](https://docs.google.com/spreadsheets/d/1BqKpkuf1-IYgqqSscOuPHwMpWlbKnMvjOlZHm92ezk8/edit?usp=sharing) over multiple quarters

**Comment by ryscher** [comment]
- Sponsorship settings and permissions have been updated. 
- Calculation of the tiered billing has been updated to prevent the problems caused by sorting issues
